src/main/java/com/team3/central/controllers/EventsApiImpl.java

```diff
@@ -110,21 +115,22 @@ public ResponseEntity<EventWithPlaces> getEventById(
    */
   @Override
   public ResponseEntity<List<Event>> getEvents() {
-    return new ResponseEntity<>(eventService.getAllEvents(),HttpStatus.OK);
+    return new ResponseEntity<>(eventService.getAllEvents(), HttpStatus.OK);
   }
 
   /**
-   *
    * User needs to be authorized
-   *
+   * <p>
```

Comment:
?

Reply:
all of the above: applying code style does this

Reply:
Revert please

Reply:
ok